### ISSUE TITLE
The latest 9205 exe installation package reports an error after running

### ISSUE THREAD

**Issue by fuzhengqu** [AlanFinotty/AAFDCHDriverMod#131]
![1](https://user-images.githubusercontent.com/5155534/126592922-0a32e56d-317d-43dd-9ddd-46dbd6cecd64.png)
An error message is displayed, "Windows cannot find setup.exe".But it can be installed normally by running the setup.exe after decompressing it by the compression software

**Comment by fuzhengqu** [comment]
The historical version does not have this problem. The first time I encountered it, there was no change in the same computer.

**Comment by M-a-e-l** [comment]
Same here 😔

**Comment by iammarxg** [comment]
Yeah the issue is most likely just a typo in the code of the installer

**Comment by AlanFinotty** [comment]
Package Reuploaded
